scripts/run_lighthouse_tests.py

```diff
@@ -90,6 +112,11 @@ def run_lighthouse_puppeteer_script() -> None:
             # print it.
             export_url(line.decode('utf-8'))
         print('Puppeteer script completed successfully.')
+        if record:
+            if os.path.isfile(video_path):
+                print('Resulting puppeteer video saved at %s' % video_path)
+            else:
+                print('No video found at %s' % video_path)
```

Comment:
My thinking was that you could have the console output whether the video was wrong or not, but in retrospect, I think making it simpler and assuming that the video has been printed is a better idea. That way, if the video was not actually created, then we could see the warning that appears when the GitHub workflow would try to upload an invalid file after the lighthouse script is run. 

Since I removed this feature, I also deleted the extra tests there were added to `run_lighthouse_tests_test.py` that were testing the case where the video also failed.